src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/CollectionRuleOptionsTests.cs

```diff
@@ -146,7 +146,6 @@ public Task CollectionRuleOptions_EventCounterTrigger_RoundTrip()
             const double ExpectedGreaterThan = 0.5;
             const double ExpectedLessThan = 0.75;
             TimeSpan ExpectedDuration = TimeSpan.FromSeconds(30);
-            const int ExpectedFrequency = 5;
```

Comment:
Note that metricsRefreshInterval is still in the test api, but it's simply ignored.

Reply:
That can be fixed after these are merged.